doc/python/webgl-vs-svg.md

```diff
@@ -20,23 +20,32 @@ jupyter:
     name: python
     nbconvert_exporter: python
     pygments_lexer: ipython3
-    version: 3.10.11
+    version: 3.11.10
   plotly:
-    description: Using WebGL for increased speed, improved interactivity, and the
-      ability to plot even more data!
+    description: Using WebGL and NumPy and Pandas for increased speed, improved interactivity,
+      and the ability to plot even more data!
     display_as: basic
     language: python
     layout: base
-    name: WebGL vs SVG
+    name: High Performance Visualization
     order: 14
-    permalink: python/webgl-vs-svg/
-    redirect_from: python/compare-webgl-svg/
+    permalink: python/performance/
+    redirect_from: python/webgl-vs-svg/
     thumbnail: thumbnail/webgl.jpg
 ---
 
-### SVG and canvas/WebGL: two browser capabilities for rendering
+For improved performance, consider using WebGL-based traces and Pandas or NumPy objects when creating figures in Plotly.
+
+<!-- #region -->
+## WebGL
+
+`plotly` figures are rendered by web browsers, which broadly speaking have two families of capabilities for rendering graphics:
+
+- The SVG API, which supports vector rendering
+- The Canvas API, which supports raster rendering, and can exploit GPU hardware acceleration via a browser technology known as WebGL.
 
-`plotly` figures are rendered by web browsers, which broadly speaking have two families of capabilities for rendering graphics: the SVG API which supports vector rendering, and the Canvas API which supports raster rendering, and can exploit GPU hardware acceleration via a browser technology known as WebGL. Each `plotly` trace type is primarily rendered with either SVG or WebGL, although WebGL-powered traces also use some SVG. The following trace types use WebGL for part or all of the rendering:
+
+Each `plotly` trace type is primarily rendered with either SVG or WebGL, although WebGL-powered traces also use some SVG. The following trace types use WebGL for part or all of the rendering:
```

Comment:
This is not correct.
Each trace is either SVG or WebGL.

Reply:
Ah okay. This is not new content. It's what we currently say on the docs. Will remove.